www/src/components/CodeEditor/CodeEditor.module.scss

```diff
@@ -17,6 +17,22 @@
   @include breakpoint("large-") {
     margin-bottom: 20px;
   }
+
+  h3 {
```

Comment:
> Are we avoiding tabs at this stage?

Tabs for what? For the code editor? I didn't think we'd discussed them. The only tabs discussion I remember was about the home page content, which we _could_ try but it's harder b'c of the janky markdown format that's currently being used. That might be a nice thing to add later, after initial release.